active/checkboxes/checkbox.md

```diff
@@ -0,0 +1,71 @@
+# Checkbox detailed design spec / redline
+This spec lists out all the details of checkbox design that should be updated as part of [#839](https://github.com/microsoft/microsoft-ui-xaml/issues/839). Depending on schedule and other unforseen circumstances, not all could be implemented so this does not represent a final plan yet.
+
+## The anatomy of checkbox
+![anatomy](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/chigy/checkboxes/active/checkboxes/Images/Anatomy.png)
+### Basic checkbox UI elements
+Checkbox has the following UI elements that determines visuals:
+1. The box fill
+2. The box border
+3. The glyph
+4. The indeterminate marker
+5. Text
+
+### Checkbox colors
+Following colors are used to denote the state of the checkbox:
+1. Fill color of the box
+2. Color of the border
+3. Foreground color of the glyph
+4. The fill color
```

Comment:
It's not clear to me fill color of what. All the other line items specify of "the box", "the glyph", etc.

Reply:
```suggestion
4. The fill color of the indeterminate indicator
```

Reply:
@YuliKl , I wasn't sure how to call it... Does this sound OK? "The fill color of the indeterminate indicator" Didn't want call it a box since that assumes the visual will always be the same.

Reply:
Sounds great.